aip/client-libraries/4221.md

```diff
@@ -0,0 +1,80 @@
+---
+aip:
+  id: 4221
+  scope: client-libraries
+  state: draft
+  created: 2019-06-17
+permalink: /client-libraries/4221
+redirect_from:
+  - /4221
+---
+
+# Client-side retry
+
+Many APIs have error modes that a client should retry. These error modes vary
+across APIs, leaving users to hand-write retry logic around client libraries
+based on available documentation.
+
```

Comment:
Add a sentence here saying, essentially, "Client libraries can make this better by automatically retrying in common, well-understood situations."

Reply:
Done.